src/Wordpress/DataStore.php

```diff
@@ -9,6 +9,7 @@ class DataStore implements DataStoreInterface
 {
     const API_KEY = 'cloudflare_api_key';
     const EMAIL = 'cloudflare_api_email';
+    const IP_REWRITE = 'cloudflare_ip_rewrite';
```

Comment:
We don't want to add a new const every time we add a setting so lets do this instead.
```
const CLOUDFLARE_SETTING_PREFIX = "cloudflare_";
const IP_REWRITE = "ip_rewrite";
```